cabra/src/pages/hooks/useTaskDetails.hook.ts

```diff
@@ -0,0 +1,50 @@
+import { deleteTask, getTask, getTasks, updateTask } from "../../api/tasks";
```

Comment:
❤️ that you put the logic into this custom hook.

Nitpick: the `use` in the file name already says that there's a hook inside, so I don't we need the `.hook` infix.